style.css

```diff
@@ -111,3 +112,269 @@ body {
   transition: 1s ease-in-out;
   cursor: pointer;
 }
+
+.works-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  gap: 88px;
+}
```

Comment:
- [ ] Great work building your Works section. 💯 However, please kindly ensure to layout the elements inside this container using display `grid` instead of display `flex` as requested in the requirement below. 👇🏽
>In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.